detectron2/export/patch_instance.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import importlib.util
+import os
+import sys
+import tempfile
+import torch
+
+from detectron2.structures import Boxes, Instances  # noqa F401
+
+_counter = 0  # TODO replace by a hash?
```

Comment:
maybe the comment isn't needed. I feel that a counter might be good enough and not sure whether a hash will be better in any way

Reply:
done, I feel the use of `counter` is good enough currently.